dts/riscv/ite/it8801-common-cfg.dtsi

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <ite/it8801-mfd-map.dtsi>
+
+&it8801_i2c {
+	it8801_mfd: it8801@38 {
+		compatible = "ite,it8801-mfd";
+		reg = <0x38>;
+		irq-gpios = <&gpioa 1 0>;
```

Comment:
this should probably not be here? I'd imagine you can connect it to any pin

Reply:
Removed.